packages/core/upload/admin/src/components/FolderCard/FolderCard/FolderCard.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { pxToRem } from '@strapi/helper-plugin';
+import { Box } from '@strapi/design-system/Box';
+import { Stack } from '@strapi/design-system/Stack';
+import Folder from '@strapi/icons/Folder';
+
+import { FolderCardContext } from '../contexts/FolderCard';
+import useId from '../hooks/useId';
+
+const FauxClickWrapper = styled.button`
+  height: 100%;
+  left: 0;
+  position: absolute;
+  opacity: 0;
+  top: 0;
+  width: 100%;
+
+  &:hover,
+  &:focus {
+    text-decoration: none;
+  }
+`;
+
+const StyledFolder = styled(Folder)`
+  path {
+    fill: currentColor;
+  }
+`;
+
+export const FolderCard = ({ children, id, startAction, ariaLabel, onDoubleClick, ...props }) => {
+  const generatedId = useId('folder', id);
```

Comment:
The useId hook only takes one parameter either so it makes me wonder if you really need this one?

Reply:
Good catch. This was a leftover and removed in 103854a3ba628bca23343e5386807fd9cb998db2